### ISSUE TITLE
Filter Spots

### ISSUE THREAD

**Issue by tischi** [BIOP/EasyXT-FIJI#20]
@romainGuiet @lacan @NicoKiaru 

Do you happen to have an example workflow for (i) filtering Spots objects based on some measurements and then (ii) creating new Spots (based on the filter) and adding them to the Scene?

**Comment by romainGuiet** [comment]
Is [MakeAndGetSpotsDemo.java](https://github.com/BIOP/EasyXT-FIJI/blob/master/src/main/java/ch/epfl/biop/imaris/demo/MakeAndGetSpotsDemo.java) what you are looking for ?

**Comment by tischi** [comment]
Not quite, I would like to filter Spots that _have been detected already_ and create a second filtered Spots instance.
Like in CellProfiler: `FilterObjects`
Is that possible?

**Comment by romainGuiet** [comment]
It is "possible", please find [here a small script](https://gist.github.com/romainGuiet/08e60992a286d7c3e44ad9f453ff0d0a) that filters the spots after the creation, using basic information of the spots ( xyz , t).
For more advance measurements filtering, one will have to get the Statistics first and use the same Logic, using Ids.

Would you prefer that EasyXT have such function taking care of this task rather than using the "build-in" filters in the SpotDetection?
